app/assets/javascripts/dashboard/study_level_activities.js.coffee

```diff
@@ -105,3 +96,12 @@ $ ->
       $.ajax
         type: 'DELETE'
         url: "/dashboard/fulfillments/#{fulfillment_id}"
+
+  $(document).on 'load-success.bs.table', '.study_level_activities', ->
+    setup_xeditable_fields()
+
+  $('body').tooltip(
```

Comment:
This is a good idea. At some point we should streamline our tooltips to do this using a global JS function.

Reply:
Yeah. The only reason I came across this was because tooltips and x-editable fields don't play nicely. Reason being the `data: { toggle: 'tooltip' }` part